### PR TITLE
Added The Wave project

### PR DESCRIPTION
**Pull request by Cast324**

### CODE REVIEW

File: thewave.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>AR - The Wave.</title>
```
Comment (SirArkimedes) [inline thewave.html]: I take it this couldn't be an emoji?

Also, could you remove the period?

File: thewave.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+				  </button>
+				  <a class="navbar-brand" href="index.html">Andrew Robinson</a>
+				</div>
+				<div id="navbar" class="navbar-collapse collapse">
+					<ul class="nav navbar-nav navbar-right">
+					<li><a href="index.html"><span class="glyphicon glyphicon-home" aria-hidden="true"></span><b> Home</b></a></li>
+					<li class="active"><a href="projects.html"><span class="glyphicon glyphicon-phone" aria-hidden="true"></span><b> Projects</b></a></li>
+					<li><a href="index.html#about"><span class="glyphicon glyphicon-user" aria-hidden="true"></span><b> About</b></a></li>
+					<li><a href="index.html#contact"><span class="glyphicon glyphicon-envelope" aria-hidden="true"></span><b> Contact</b></a></li>
+<!--					<li class="dropdown">
```
Comment (SirArkimedes) [inline thewave.html:42]: Can you remove this comment, please?

**Review by SirArkimedes — CHANGES_REQUESTED** [review]
There are 5 screenshots in the repo. Will you add them all in the same order, please and thanks?

Also, could you put The 👋 as the first project on the home page carousel?

**Comment by Cast324** [comment]
Updated branch

**Comment by SirArkimedes** [comment]
Thanks! 👋

**Pull request merged**